Makefile.am

```diff
@@ -202,6 +202,21 @@ ci: rc-release
 	@sudo rm -f $(ROOT_SITE_PACKAGES)/usercustomize.py
 	$(MAKE) coverage-report
 
+test-gui:
+	@rm -f tests/test-suite.log
+	@rm -rf tests/autogui-results-*/
+	@sudo mkdir -p $(ROOT_SITE_PACKAGES)
+	@sudo cp $(abs_builddir)/tests/usercustomize.py $(ROOT_SITE_PACKAGES)
+# We require SELinux in Permissive mode, see rhbz#1276376
+	@sudo setenforce 0
+	@sudo COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc \
+		$(abs_builddir)/tests/run_gui_tests.sh >tests/test-suite.log 2>&1
+	@sudo setenforce 1
```

Comment:
OK, I agree I might turn selinux on if it was turned off. But Dogtail tests will not run of I don't call setenforce 0. So I will simply restore whatever SELinux state was there previously.